packages/local_auth/local_auth_windows/pubspec.yaml

```diff
@@ -0,0 +1,27 @@
+name: local_auth_windows
+description: Windows implementation of the local_auth plugin.
+repository: https://github.com/flutter/plugins/tree/master/packages/local_auth/local_auth_windows
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+local_auth%22
+version: 1.0.0
+
+environment:
+  sdk: ">=2.14.0 <3.0.0"
+  flutter: ">=2.8.0"
+
+flutter:
+  plugin:
+    implements: local_auth
+    platforms:
+      windows:
+        pluginClass: LocalAuthPlugin
+        dartPluginClass: LocalAuthWindows
+
+dependencies:
+  flutter:
+    sdk: flutter
+  intl: ^0.17.0
```

Comment:
Which means you can drop this dependency.

Reply:
Fixed.